docs/concepts/rdf.md

```diff
@@ -0,0 +1,55 @@
+# RDF in an Islandora Repository
+
+Islandora was build on the idea that a repository can be represented
+in Drupal in a way that can be mapped to RDF and present the repository 
+as Linked Data.
+
+## Portland Common Data Model
+
+Islandora uses the [Portland Common Data Model (PCDM)](https://github.com/duraspace/pcdm/wiki)
+ to arrange elements
+of the repository. Nodes correspond to pcdm:Objects, and Media to pcdm:Files.
+The "Member of" field on nodes allows us to create pcdm:memberOf relationships
```

Comment:
Checking - should this stay "Member of" or be referred to as `field_member_of` here?

Reply:
I'm pretty happy leaving this in "user language" in this place.